README.md

```diff
@@ -95,3 +95,15 @@ It provides a dashboard for Kubernetes and ks-devops.
 
 ## License
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2FLinuxSuRen%2Fks-devops.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2FLinuxSuRen%2Fks-devops?ref=badge_large)
+
+## Available communication channels:
+
+- [kubesphere Devops google group ](https://groups.google.com/g/kubesphere-sig-devops/)
```

Comment:
`kubesphere` -> `KubeSphere`

I prefer to remove the extra space here if you don't have a strong reason for that.

`[kubesphere Devops google group ]` -> `[kubesphere Devops google group]`